integrations/googledrive/src/file-notification-token.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from '@botpress/sdk'
+import { fileTypesUnionSchema } from './schemas'
+
+const tokenSchema = z.object({
+  fileId: z.string().min(1),
+  fileType: fileTypesUnionSchema,
+})
+export type Token = z.infer<typeof tokenSchema>
+
+export const serializeToken = (token: Token) => {
```

Comment:
**chore:** use an integration secret to sign tokens with JWT

Reply:
Will be implemented in a separate PR